website_version_ce/static/src/js/menu_version.js

```diff
@@ -0,0 +1,294 @@
+                self.$el.find(".o_experiment").remove();
+                if(result){
+                    self.$el.find(".create_experiment").after(qweb.render("website_version_ce.experiment_menu"));
+                }
+            });
+
+        });
+        return this._super();
+    },
+
+    duplicate_version: function(event) {
+        var version_id = base.get_context().version_id;
+        var wizardA = $(qweb.render("website_version_ce.new_version",{'default_name': moment().format('L')}));
+        wizardA.appendTo($('body')).modal({"keyboard" :true});
+        wizardA.on('click','.o_create', function(){
+            wizardA.find('.o_message').remove();
+            var version_name = wizardA.find('.o_version_name').val();
+            if(version_name.length === 0){
+                wizardA.find(".o_version_name").after("<p class='o_message' style='color : red'> *"+_t("This field is required")+"</p>");
+            }
+            else{
+                wizardA.modal("hide");
+                ajax.jsonRpc( '/website_version_ce/create_version', 'call', { 'name': version_name, 'version_id': version_id}).then(function (result) {
+
+                    var wizard = $(qweb.render("website_version_ce.dialogue",{message:_.str.sprintf("You are now working on version: %s.", version_name),
+                                                                                   dialogue:_.str.sprintf("If you edit this page or others, all changes will be recorded in the version. It will not be visible by visitors until you publish the version.")}));
+                    wizard.appendTo($('body')).modal({"keyboard" :true});
+                    wizard.on('click','.o_confirm', function(){
+                        window.location.href = '\?enable_editor';
+                    });
+                    wizard.on('hidden.bs.modal', function () {$(this).remove();});
+                }).fail(function(){
+                    var wizard = $(qweb.render("website_version_ce.message",{message:_t("This name already exists.")}));
+                    wizard.addClass("o_error");
+                    wizard.appendTo($('body')).modal({"keyboard" :true});
+                    wizard.on('hidden.bs.modal', function () {$(this).remove();});
+                });
+            }
+        });
+        wizardA.on('hidden.bs.modal', function () {$(this).remove();});
+    },
+
+    change_version: function(event) {
+        var version_id = parseInt($(event.target).closest("li").data("version_id"));
+        if(! version_id){
+            version_id = 0;//By default master
+        }
+        ajax.jsonRpc( '/website_version_ce/change_version', 'call', { 'version_id':version_id }).then(function (result) {
+                location.reload();
+            });
+    },
+
+    delete_version: function(event) {
+        var version_id = parseInt($(event.target).closest("li").data("version_id"));
```

Comment:
Actually, there is a difference between event.target and event.currentTarget.

Given the stackoverflow link below, and seeing how the listener might be the whole widget (see line 24 of menu_version.js), I'm thinking the uses of target and currentTarget are very intentional.

See here:http://stackoverflow.com/questions/5921413/difference-between-e-target-and-e-currenttarget